src/Sponsorship/Resources/config/services.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+    bind:
+      $currency: '%currency%'
```

Comment:
Not a fan of that bind `$currency` as it prevents name changes within a class... That's another issue